### DISCUSSION TITLE
bash script deployment

### DISCUSSION THREAD

**Discussion by lephtHanded** [SquirrelCorporation/SquirrelServersManager#225] [Q&A]
I'm looking for something that I can centrally manage and deploy bash scripts to my servers.  
Can the playbooks feature be leveraged to run shell scripts remotely?

**Comment by SquirrelDeveloper** [comment]
Hi @lephtHanded Absolutely, you can either migrate your scripts to ansible commands https://docs.ansible.com/ansible/latest/collections/ansible/builtin/shell_module.html or directly uses your files with https://docs.ansible.com/ansible/latest/collections/ansible/builtin/script_module.html
SSM provides way of adding either Github repositories or local repositories as Playbooks Repositories in the settings

**Reply by lephtHanded** [reply]
thanks!
I'll take a look.

**Reply by SquirrelDeveloper** [reply]
No problem, dont hesitate
